modules/serverless-rn-1-25-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies
+//
+// * serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-25-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.25.0
+
+{ServerlessProductName} 1.25.0 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1.25.0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.4.
+* {ServerlessProductName} now uses Knative Eventing 1.4.
+* {ServerlessProductName} now uses Kourier 1.4.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.4.
+* {ServerlessProductName} now uses Knative Kafka 1.4.
+* The `kn func` CLI plug-in now uses `func` 1.7.0.
```

Comment:
```suggestion
* The `kn func` CLI plug-in now uses Knative Functions 1.7.0.
```

Reply:
@abrennan89 I've changed this per Lance's suggestion. Seems like GA is a good time to `s/func/Knative Functions/`. Please let me know if I should revert back to "`func`".

Reply:
lgtm